<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveDuplicateDialog</class>
 <widget class="QDialog" name="RemoveDuplicateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>去除重复项</string>
  </property>
  <layout class="QVBoxLayout" name="rootLayout" stretch="0,1,0,0">
   <item>
    <widget class="OtoFileLoadWidget" name="otoLoadWidget" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="optionGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>行为设定</string>
     </property>
     <layout class="QVBoxLayout" name="optionLayout">
      <item>
       <widget class="QWidget" name="optionWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="OtoFileSaveWidget" name="otoSaveWidget" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OtoFileLoadWidget</class>
   <extends>QWidget</extends>
   <header>utils/widgets/otofileloadwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OtoFileSaveWidget</class>
   <extends>QWidget</extends>
   <header>utils/widgets/otofilesavewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemoveDuplicateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemoveDuplicateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
